sort.java

```diff
@@ -7,7 +7,7 @@ static void bubbleSort(int arr[], int n)
         for (int i=0; i<n-1; i++){
```

Comment:
Suggest using java streams to iterating over the array instead of traditional for loop